File: server/services/boletos.api/models/classes/boleto.js
class Boleto{
    constructor(boleto){
        this.no_boleto = boleto.no_boleto || null;
        this.no_operacion = boleto.no_operacion || null;
        this.cve_asiento = boleto.cve_asiento || null;
        this.cve_estado = boleto.cve_estado || 1 || null;
        this.nombre_pas = boleto.nombre_pas || null;
        this.token_fac = boleto.token_fac || null;
        this.no_asiento_boleto = boleto.no_asiento_boleto || null;
        this.puerta = boleto.puerta || null;
        this.carril = boleto.carril || null;
        this.anden = boleto.anden || null;
        this.tel_cliente = boleto.tel_cliente || null;
        this.costo = boleto.costo || null;
    }
}

module.exports = Boleto;

File: app-legacy/Flecha Digital/Model/Boleto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Flecha_Digital.Model
{
    public class Boleto
    {
        public int no_boleto { get; set; }
        public int cve_tipo { get; set; }
        public int cve_asiento { get; set; }
        public string nombre_pas { get; set; }
        public string token_fac { get; set; }
        public int no_asiento_boleto { get; set; }
        //public string puerto {get; set;}
        //public int carril {get; set;}
        //public int anden {get; set;}
        public int tel_cliente { get; set; }
        public double costo_boletos { get; set; }
    }
}